src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -1809,6 +1809,21 @@ public override void Accept(OperationVisitor visitor)
         }
     }
 
+    internal partial class BoundMethodToken
+    {
+        protected override OperationKind ExpressionKind => OperationKind.None;
+
+        public override void Accept(OperationVisitor visitor)
```

Comment:
All of the other unexposed node types act this way. A systematic change to all of them might be worthwhile, but is outside the scope of this change set.